cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KRaftUtils.java

```diff
@@ -24,14 +23,11 @@ public class KRaftUtils {
      * unsupported features and if they are used, throws an InvalidResourceException exception.
      *
      * @param kafkaSpec   The .spec section of the Kafka CR which should be checked
-     * @param utoEnabled  Flag indicating whether Unidirectional Topic Operator is enabled or not
      */
-    public static void validateKafkaCrForKRaft(KafkaSpec kafkaSpec, boolean utoEnabled)   {
+    public static void validateKafkaCrForKRaft(KafkaSpec kafkaSpec)   {
         Set<String> errors = new HashSet<>(0);
 
-        if (kafkaSpec != null)  {
-            validateEntityOperatorSpec(errors, kafkaSpec.getEntityOperator(), utoEnabled);
-        } else {
+        if (kafkaSpec == null)  {
```

Comment:
I can't remember (@scholzj added this so he should know) but maybe this validation method was added for checking that UTO was used when the cluster is KRaft. Now that UTO is the only choice and BTO is gone, who much does it make sense having a validation about `.spec` not null in this `KRaftUtils` class? I mean, it's a validation which makes sense for any cluster, ZooKeeper or KRaft based. Wdyt?

Reply:
Sorry, I'm not following what you mean. Where do you want to move this check?

Reply:
I was just saying that checking for the `spec` not being null is not just for KRaft clusters but even for ZooKeeper-based clusters.
Anyway I saw we have a `validateKafkaCrForZooKeeper` which runs the same validation and it's still in the `KRaftUtils` class ... even if it's not KRaft related stuff. So maybe it's misleading.
At this point, nothing to do from your side.